src/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/repair/Sorald.java

```diff
@@ -62,17 +62,18 @@ protected StepStatus businessExecute() {
         for (String rule : RepairnatorConfig.getInstance().getSonarRules()) {
             this.getLogger().info("Repo: " + pathToRepoDir);
             this.getLogger().info("Try to repair rule " + rule);
-
-            try {
-                Main.main(new String[]{
+            String[] args = new String[]{
                             "--originalFilesPath",pathToRepoDir,
                             "--ruleKeys",rule,
-                            "--workspace",RepairnatorConfig.getInstance().getWorkspacePath(),
+                            "--workspace", this.getConfig().getWorkspacePath(),
                             "--gitRepoPath",pathToRepoDir,
                             "--prettyPrintingStrategy","SNIPER",
-                            "--maxFixesPerRule","1"});
+                            "--maxFixesPerRule","" + getConfig().getSoraldMaxFixesPerRule(),
+                            "--repairStrategy",RepairnatorConfig.getInstance().getSoraldRepairMode().name(),
+                            "--maxFilesPerSegment","" + RepairnatorConfig.getInstance().getSegmentSize()};
+            try {
+                Main.main(args);
             } catch(Exception e) {
-                e.printStackTrace();
```

Comment:
use proper call to the logging library

Reply:
`e.printStackTrace()` is already replaced by `StepStatus.buildSkipped(..)` in the subsequent line.